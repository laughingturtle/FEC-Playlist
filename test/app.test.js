import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../client/index.jsx';


describe('<App />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const component = shallow(<App />);
      //const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    })
  })
});
