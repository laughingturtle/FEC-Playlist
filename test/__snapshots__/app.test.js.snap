// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render() renders the component 1`] = `
<div>
  <styled.section>
    <styled.h1>
      Songs Library
    </styled.h1>
  </styled.section>
  <styled.button
    onClick={[Function]}
  >
    Home
  </styled.button>
  <div>
    <Grid
      bsClass="container"
      componentClass="div"
      fluid={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/M1kn0Yt/hiphop.png"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/JmtzL59/country.jpg"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/r3h1vrp/classical.jpg"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/2Np9g4g/jazz.jpg"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/XXMjhLn/pop.jpg"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
        <Col
          bsClass="col"
          componentClass="div"
          md={2}
          xs={3}
        >
          <Image
            alt="171x180"
            bsClass="img"
            circle={false}
            onClick={[Function]}
            responsive={false}
            rounded={false}
            src="https://i.ibb.co/23jKYxF/rock.jpg"
            style={
              Object {
                "height": 200,
                "width": 200,
              }
            }
            thumbnail={true}
          />
        </Col>
      </Row>
    </Grid>
    <br />
    <AudioList
      audio={Array []}
    />
  </div>
</div>
`;
